command/operator_migrate.go

```diff
@@ -84,6 +84,7 @@ func (c *OperatorMigrateCommand) Flags() *FlagSets {
 
 	f.StringVar(&StringVar{
 		Name:   "config",
+		EnvVar: "BAO_OP_MIGRATE_CONFIG_PATH",
```

Comment:
I think I'd go for

```suggestion
		EnvVar: "BAO_OPERATOR_MIGRATE_CONFIG_PATH",
```

(consistent version)

or even just


```suggestion
		EnvVar: "BAO_MIGRATE_CONFIG_PATH",
```

(simple version)

Thoughts?

Reply:
I also think BAO_MIGRATE_CONFIG_PATH looks better, so I’ll go ahead with that.